<launch>
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find map_generator)/rviz/display_grid_map_2d.rviz"/>

    <node pkg="map_generator" type="map_generator_2d" name="map_generator_2d" output="screen">
        <param name="init_x" value="0"/>
        <param name="init_y" value="0"/>

        <param name="map/x_size" value="10"/>
        <param name="map/y_size" value="10"/>

        <param name="Obstacle/lower_w" value="0.3"/>
        <param name="Obstacle/upper_w" value="1.0"/>

        <param name="map/obstacle_number" value="100"/>
        <param name="map/resolution" value="0.05"/>

        <param name="sensor/rate" value="0.5"/>
    </node>
</launch>